scripts/_resources/r_god.gd

```diff
@@ -1,14 +1,14 @@
 extends Resource
 class_name RGod
 
-@export var image: Image = null
+@export var image: Texture2D = null
 @export var passive: String = ""
-@export var favor: RMaterialStack = null
-@export var tithe: RMaterialStack = null
+@export var tithe: RTithe = null
+@export var favor_cap: int = 0
```

Comment:
I dont think favor cap should be in the god.  If it really should, then it should be a materialstack rather than a float to allow us to couple it to the actual material.

Reply:
I changed it to be in a resourced called [GOD_NAME}_favor_cap.tres as a MaterialStack. I also changed the one generic favor material to a cat-specific material and create a crow and pig one as well, and set them as the materials with their respective favor caps. I am loving this Resource composition stuff.